downstream/modules/eda/con-eda-rulebook-activation-list-view.adoc

```diff
@@ -0,0 +1,21 @@
+[id="eda-rulebook-activation-list-view"]
+
+= Rulebook activation list view
+
+On the *Rulebooks Activations* page you can view the rulebooks that you have created along with the *Activation status*, *Number of rules associated* with the rulebook, the *Fire count* and *Restart count*.
+
+If the *Activation Status* is *Running*, it means that the rulebook activation or rulebook is constantly running in the background and executing the required action.
+
+You can view more details by selecting the activation from the *Rulebook Activations* list view.
+
+image::eda-rulebook-activations-list-view.png[Rulebook activation][width=25px]
+
+For example, if your activation has failed you can see the *Details* and *History* tabs.
```

Comment:
Minor: This sentence suggest that you can only see the details and history only if the activation has failed, but you can see them always. To be clearer I would say:
For example, if your activation has failed you can see the *Details* and *History* tabs to get more info about what happened.

Reply:
Would it potentially be less confusing if it were more generic? 
Ex:
For all activations that have run, you can see the Details and History tabs to get more information about what happened.

Reply:
yeah, it is another approach. I think the main goal here is provide a guide to get the output of the activations, which is not obvious and is something that you usually want to do in case of errors. I'm thinking that it could be a different subsection, but I'm not the expert of docs writing :)

Reply:
Thanks for the further info here, I agree with you Alejandro and have created a subsection **5.2.1. Viewing activations output**, please take a look a let me know if this works.